docker/package/packages.py

```diff
@@ -390,6 +390,7 @@ def mk_node_unit(
             additional_native_deps=[
                 "tezos-sapling-params",
                 "tezos-client",
+                "tezos-node",
```

Comment:
This seems to me the wrong place to add this, as this is the `tezos-baker-*` package definition.

One should be able to install the baker without also installing a node (it's not a strict dependency for it).

OTOH this dependency should be part of the `tezos-baking` package instead

Reply:
why then it makes sense to add `tezos-client` as dependency?

Reply:
because `tezos-client` is necessary to register a baker, before one can run the baker process.